### PR TITLE
Added missing license and copyright lines

### PR DESCRIPTION
**Pull request by rohitmohan96**
Here's the output of `reuse lint`: 

```
# MISSING COPYRIGHT AND LICENSING INFORMATION

The following files have no copyright and licensing information:
* .gitignore
* gateway/.dockerignore
* gateway/.gitignore
* gateway/certs/tiedie-ca-openssl.cnf
* gateway/proto/data_app_pb2.py
* gateway/proto/data_app_pb2.pyi
* gateway/requirements.txt
* gateway/silabs/api/sl_bt.xapi
* gateway/silabs/api/sl_btmesh.xapi
* gateway/silabs/common/libcpc_wrapper.py
* java-sdk/.idea/gradle.xml
* java-sdk/.idea/inspectionProfiles/Project_Default.xml
* java-sdk/.idea/markdown.xml
* java-sdk/.idea/misc.xml
* java-sdk/.idea/uiDesigner.xml
* java-sdk/.idea/vcs.xml
* java-sdk/.idea/workspace.xml
* java-sdk/sample-java-app/.gitignore
* java-sdk/sample-java-app/config/application.properties
* java-sdk/sample-java-app/gradle/wrapper/gradle-wrapper.jar
* java-sdk/sample-java-app/gradle/wrapper/gradle-wrapper.properties
* java-sdk/sample-java-app/src/main/resources/META-INF/additional-spring-configuration-metadata.json
* java-sdk/sample-java-app/src/main/resources/application.properties
* java-sdk/sdk/.gitignore
* java-sdk/sdk/gradle/wrapper/gradle-wrapper.jar
* java-sdk/sdk/gradle/wrapper/gradle-wrapper.properties
* java-sdk/sdk/lombok.config
* java-sdk/sdk/settings.gradle
* python-sdk/requirements.txt
* python-sdk/sample-python-app/config/config.ini
* python-sdk/sample-python-app/requirements.txt
* python-sdk/tiedie_pylib/api/proto/data_app_pb2.py
* python-sdk/tiedie_pylib/api/proto/data_app_pb2.pyi

The following files have no licensing information:
* java-sdk/sample-java-app/gradlew
* java-sdk/sample-java-app/gradlew.bat
* java-sdk/sdk/gradlew
* java-sdk/sdk/gradlew.bat


# SUMMARY

* Bad licenses: 0
* Deprecated licenses: 0
* Licenses without file extension: 0
* Missing licenses: Apache-2.0, Zlib
* Unused licenses: 0
* Used licenses: Zlib, Apache-2.0
* Read errors: 0
* files with copyright information: 154 / 187
* files with license information: 150 / 187
```

I'm not sure if these misc files can be licensed.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<!--
+Copyright (c) 2023, Cisco and/or its affiliates.
+All rights reserved.
+See LICENSE file in this distribution.
+SPDX-License-Identifier: Apache-2.0
+-->
```
Comment (elear) [inline CODE_OF_CONDUCT.md:6]: XML comment for .md?  Not sure this is correct (same directly below).

File: gateway/db/init.sql [hunk 2]
```diff
@@ -1,2 +1,7 @@
+-- Copyright (c) 2023, Cisco and/or its affiliates.
+-- All rights reserved.
+-- See LICENSE file in this distribution.
+-- SPDX-License-Identifier: Apache-2.0
+
 CREATE DATABASE tiedie;
```
Comment (elear) [inline gateway/db/init.sql:7]: have you tested this?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (rohitmohan96) [reply CODE_OF_CONDUCT.md:6]: Markdown allows inline HTML. 

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#hiding-content-with-comments

File: gateway/db/init.sql (hunk 2 above)
Comment (rohitmohan96) [reply gateway/db/init.sql:7]: Yes, I tested it. That is a valid SQL comment.

**Comment by elear** [comment]
Before we merge, can you create a parallel PR for the ios branch?  Easiest way I've found is to diff what you have against main, checkout the ios branch, patch, resolve the one or to .rejs and PR.

**Comment by rohitmohan96** [comment]
Created #22

**Review by elear — APPROVED** [review]

**Pull request merged**